Meta/Lagom/CMakeLists.txt

```diff
@@ -143,6 +143,11 @@ if (ENABLE_LAGOM_LADYBIRD AND (ENABLE_FUZZERS OR ENABLE_COMPILER_EXPLORER_BUILD)
     )
 endif()
 
+CHECK_INCLUDE_FILE(pulse/pulseaudio.h HAVE_PULSEAUDIO)
+if (HAVE_PULSEAUDIO)
+    add_compile_definitions(HAVE_PULSEAUDIO=1)
```

Comment:
Does everyone need to know this? Or could we add the compile definition to the LibAudio target only?

```
target_compile_definitions(LibAudio PRIVATE HAVE_PULSEAUDIO=1)
```
in the if (HAVE_PULSEAUDIO) block in its CMakeLists.txt

Reply:
Ahh ok, WebContent needs to know as well. Hm. Even still, would be nice to just add it to the targets  that care about it.

Reply:
Seems reasonable to me, I'll put it in Ladybird's WebContent and LibAudio only. Eventually we should be able to delete it from WebContent, if we have enough implementations that people aren't concerned about deleting the Qt plugin.